File: pbweberog/pbweberog-jar/src/main/java/it/csi/pbandi/pbweberog/integration/request/RequestAssociaIrregolarita.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbweberog.integration.request;

import it.csi.pbandi.pbservizit.pbandisrv.dto.revoca.SaveDettaglioDTO;
import it.csi.pbandi.pbservizit.pbandisrv.dto.revoca.SaveDsDTO;
import it.csi.pbandi.pbservizit.pbandisrv.dto.revoca.SaveIrregolaritaDTO;

public class RequestAssociaIrregolarita {
	private SaveIrregolaritaDTO[] elencoSaveIrregolaritaDTO; 
	private SaveDettaglioDTO[] elencoSaveDettaglioDTO;
	private SaveDsDTO[] elencoSaveDsDTO;
	private Long idRevoca ;
	public SaveIrregolaritaDTO[] getElencoSaveIrregolaritaDTO() {
		return elencoSaveIrregolaritaDTO;
	}
	public void setElencoSaveIrregolaritaDTO(SaveIrregolaritaDTO[] elencoSaveIrregolaritaDTO) {
		this.elencoSaveIrregolaritaDTO = elencoSaveIrregolaritaDTO;
	}
	public SaveDettaglioDTO[] getElencoSaveDettaglioDTO() {
		return elencoSaveDettaglioDTO;
	}
	public void setElencoSaveDettaglioDTO(SaveDettaglioDTO[] elencoSaveDettaglioDTO) {
		this.elencoSaveDettaglioDTO = elencoSaveDettaglioDTO;
	}
	public SaveDsDTO[] getElencoSaveDsDTO() {
		return elencoSaveDsDTO;
	}
	public void setElencoSaveDsDTO(SaveDsDTO[] elencoSaveDsDTO) {
		this.elencoSaveDsDTO = elencoSaveDsDTO;
	}
	public Long getIdRevoca() {
		return idRevoca;
	}
	public void setIdRevoca(Long idRevoca) {
		this.idRevoca = idRevoca;
	}
	
	

}

File: pbweberogwcl/src/app/disimpegni/commons/models/request-associa-irregolarita.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { SaveDettaglioDTO } from '../dto/save-dettaglio-dto';
import { SaveDsDTO } from '../dto/save-ds-dto';
import { SaveIrregolaritaDTO } from '../dto/save-irregolarita-dto';
export class RequestAssociaIrregolarita {
  constructor(
    public elencoSaveDettaglioDTO: Array<SaveDettaglioDTO>,
    public elencoSaveDsDTO: Array<SaveDsDTO>,
    public elencoSaveIrregolaritaDTO: Array<SaveIrregolaritaDTO>,
    public idRevoca: number
  ) { }
}
